components/pull-to-refresh.js

```diff
@@ -0,0 +1,95 @@
+import { useRouter } from 'next/router'
+import { useState, useRef, useEffect, useCallback } from 'react'
+import styles from './pull-to-refresh.module.css'
+
+const PULL_THRESHOLD = 300
```

Comment:
100 as twice the threshold when `pull to refresh` starts showing might be a better threshold.

I would also call this `REFRESH_THRESHOLD` since `PULL_THRESHOLD` sounds like that's when we show the first message.

Reply:
Noted, you're right it would make much more sense!